clients/drcachesim/tests/scheduler_unit_tests.cpp

```diff
@@ -82,7 +82,7 @@ class mock_reader_t : public reader_t {
 };
 
 static trace_entry_t
-make_instr(addr_t pc)
+make_instr(addr_t pc, memref_tid_t tid = 1)
```

Comment:
tid unused?  Probably left over from when we used memref_t?

Reply:
Yes, removed.